build/doc.go

```diff
@@ -22,11 +22,9 @@ a go development environment or using existing one for this project is not desir
 Available commands are:
 
 	install        -- install the dst-go software
-	lint           -- run meta-linter with configuration linterConfig.json
+	lint           -- run meta-linter with configuration "dst-go/.golangci.yml"
 	test           -- run unit tests, with options as in "go test" command
-	runWalkthrough -- build and run walkthrough. Behavior is configurable with
-	                  options such as "--simulated_backend", "--real_backend",
-	                  etc.
+	runWalkthrough -- build and run walkthrough, use "-h" flag to see options
```

Comment:
So, this message appears when one invokes `make help`. When I call `make -h` or `make runWalkthrough -h` as advised here, the output is
```
make runWalkthrough -h
Usage: make [options] [target] ...
Options:
  -b, -m                      Ignored for compatibility.
<snip />
```
Do you mean invoke with `BUILDOPTS="-h"` to see options?

Reply:
Yes, as it is the case for for the other options mentioned in `make help`. 
This information can be added in make help message in a follow up issue.